code/modules/mob/mob_helpers.dm

```diff
@@ -22,6 +22,27 @@
 			return 0
 	return 1
 
+/mob/proc/get_screen_colour()
+
+/mob/dead/observer/get_screen_colour()
+	return list()

```

Comment:
Straight-up gutted it and it was just fine, thankfully. No further tweaks were necessary